.changeset/orange-icons-build.md

```diff
@@ -0,0 +1,41 @@
+---
+"@cloudflare/workers-shared": minor
+"wrangler": minor
+---
+
+feat: Create very basic Asset Server Worker and plumb it into `wrangler dev`
+
+These changes do the ground work needed in order to add Assets support for Workers in `wrangler dev`. They introduce the following components, and the following conceptual "client pipeline":
+
+<!-- prettier-ignore-start -->
+                   |
+              wrangler dev
+                   |              workerd
++------------------+---------------------+
+|                  |                     |
+|      +-----------v--------------+      |
+|      |                          |      |
+|      |     NOOP Entry Worker    |      |
```

Comment:
Is this the `Router Worker`?

Reply:
not yet. We are introducing that in a separate iteration, as there are still some open questions around it

Reply:
Strictly this is not "noop" it is actually a passthrough Worker. 😄

Reply:
fair!